<div style="background:linear-gradient(135deg,#042f2e 0%,#0f766e 55%,#14b8a6 100%);border-radius:18px;padding:34px 30px;color:#fff;font-family:Inter,Segoe UI,sans-serif">
  <div style="font-size:12px;letter-spacing:3px;color:#99f6e4;font-weight:700;text-transform:uppercase">Chapter 39 · Solutions</div>
  <div style="font-size:34px;font-weight:900;line-height:1.1;margin:10px 0 6px">The Normal Distribution, Worked Answers ✅</div>
  <div style="font-size:15px;color:#d1fae5;max-width:720px;line-height:1.6">Full solutions to the five challenges. Try them yourself first, then compare.</div>
  <div style="margin-top:16px;font-size:13px;color:#a7f3d0">Statistics, Data Science and AI: A Visual Handbook · John Fisher · 2026</div>
</div>

## ⚙️ Setup

In [1]:
import numpy as np
from scipy import stats
print("ready")

ready


<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0d9488;letter-spacing:1px">CHALLENGE 1 · EMPIRICAL RULE</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Reading 68 - 95 - 99.7</div>
<div style="color:#4a5578;margin-top:6px">Adult heights are Normal(170 cm, 8 cm). Without a calculator, what fraction of people are between 162 and 178 cm? Between 154 and 186 cm?</div>
</div>

In [2]:
mu, sd = 170, 8
print(f"162 to 178 is mu +/- 1 sigma -> about 68%   (exact {(stats.norm.cdf(178,mu,sd)-stats.norm.cdf(162,mu,sd))*100:.1f}%)")
print(f"154 to 186 is mu +/- 2 sigma -> about 95%   (exact {(stats.norm.cdf(186,mu,sd)-stats.norm.cdf(154,mu,sd))*100:.1f}%)")

162 to 178 is mu +/- 1 sigma -> about 68%   (exact 68.3%)
154 to 186 is mu +/- 2 sigma -> about 95%   (exact 95.4%)


**Answer:** 162 to 178 is &#956; &plusmn; 1&#963;, so about **68%**; 154 to 186 is &#956; &plusmn; 2&#963;, so about **95%**. The empirical rule lets you answer these in your head, no z-table required.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0891b2;letter-spacing:1px">CHALLENGE 2 · z-SCORE</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">How many standard deviations?</div>
<div style="color:#4a5578;margin-top:6px">On a test with mean 75 and standard deviation 5, a student scores 85. Find the z-score and the percentile.</div>
</div>

In [3]:
z = (85 - 75) / 5
print(f"z = (85 - 75)/5 = {z:.1f}")
print(f"percentile = P(Z < 2) = {stats.norm.cdf(z)*100:.1f}%")

z = (85 - 75)/5 = 2.0
percentile = P(Z < 2) = 97.7%


**Answer:** z = (85 &minus; 75)/5 = **2.0**, so the score is two standard deviations above the mean. That places the student at the **97.7th percentile**, higher than about 98% of test-takers.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0d9488;letter-spacing:1px">CHALLENGE 3 · COMPARE VIA z</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Apples to oranges</div>
<div style="color:#4a5578;margin-top:6px">Ana scored 88 on Exam A (mean 80, sd 4). Ben scored 90 on Exam B (mean 85, sd 10). Whose result is more impressive relative to peers?</div>
</div>

In [4]:
zA = (88 - 80) / 4
zB = (90 - 85) / 10
print(f"Ana: z = {zA:.2f}")
print(f"Ben: z = {zB:.2f}")
print("Ana is stronger relative to her exam" if zA>zB else "Ben is stronger")

Ana: z = 2.00
Ben: z = 0.50
Ana is stronger relative to her exam


**Answer:** Ana's z = 2.0 versus Ben's z = 0.5. Although Ben's raw score is higher, **Ana** is far more exceptional relative to her exam (two standard deviations above the mean versus half of one). The z-score, not the raw number, is the fair comparison.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#db2777;letter-spacing:1px">CHALLENGE 4 · TAIL PROBABILITY</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">Beyond the mean</div>
<div style="color:#4a5578;margin-top:6px">IQ scores are Normal(100, 15). What fraction of people score above 130 (the 'gifted' threshold)?</div>
</div>

In [5]:
mu, sd = 100, 15
z = (130 - mu)/sd
print(f"z = (130-100)/15 = {z:.1f}")
print(f"P(X > 130) = 1 - P(Z < 2) = {(1-stats.norm.cdf(z))*100:.2f}%")

z = (130-100)/15 = 2.0
P(X > 130) = 1 - P(Z < 2) = 2.28%


**Answer:** 130 is z = 2 above the mean, so P(X &gt; 130) = 1 &minus; &Phi;(2) &approx; **2.28%**. By the empirical rule, about 2.5% lie beyond +2&#963;, matching the exact 2.28%.

<div style="background:#ecfdf5;border-left:5px solid #0d9488;border-radius:10px;padding:16px 20px;font-family:Inter,sans-serif">
<span style="font-size:13px;font-weight:800;color:#0f766e;letter-spacing:1px">CHALLENGE 5 · STANDARDIZE A DATASET</span>
<div style="font-size:21px;font-weight:800;color:#1a2138;margin-top:4px">The ML preprocessing step</div>
<div style="color:#4a5578;margin-top:6px">Standardize the small dataset [10, 12, 14, 16, 18] to z-scores (mean 0, std 1), the way a model would before training.</div>
</div>

In [6]:
x = np.array([10, 12, 14, 16, 18])
z = (x - x.mean()) / x.std()
print(f"mean = {x.mean()}, std = {x.std():.3f}")
print(f"z-scores = {z.round(3)}")
print(f"check: mean {z.mean():.3f}, std {z.std():.3f}")

mean = 14.0, std = 2.828
z-scores = [-1.414 -0.707  0.     0.707  1.414]
check: mean 0.000, std 1.000


**Answer:** the mean is 14 and the standard deviation is &#8730;8 &approx; 2.83, giving z-scores [&minus;1.41, &minus;0.71, 0, 0.71, 1.41]. The standardized data has mean 0 and std 1, exactly what `StandardScaler` produces, and the most common feature-preprocessing step in machine learning.

---
<div style="background:#ffffff;border:1px solid #e6e9f2;border-radius:16px;padding:22px 26px;font-family:Inter,sans-serif">
<div style="font-size:19px;font-weight:800;color:#1a2138">🎉 Nicely done</div>
<div style="color:#4a5578;line-height:1.8;margin-top:8px">You applied the empirical rule, computed and compared z-scores, found a tail probability, and standardized a dataset. The z-score, how many standard deviations from the mean, ties together percentiles, comparisons, anomaly detection, and ML preprocessing.</div>
</div>

<div style="text-align:center;color:#8b94b3;font-size:12px;margin-top:16px">Statistics, Data Science and AI: A Visual Handbook · © 2026 John Fisher</div>